### DISCUSSION TITLE
Why i cant using two HC-SR04 ultrasonic ranging sensor?

### DISCUSSION THREAD

**Discussion by mefisto22** [arendst/Tasmota#15635] [Q&A] [answered]
When i connect HC-SR04 ultrasonic ranging sensor to esp8266, only one works. When i unplug first sensor, second start to work.

**Comment by sfromis** [comment, marked as answer]
As designed. Tasmota only supports one such sensor. Same goes for most other sensors.

**Reply by barbudor** [reply]
On the other end, instead of crappy ultrasonic sensor, you can use up to 8 VL53L0X or VL53L1X Time-of-Flight sensors.
Much more reliable
